lib/isorender/dom-shims.js

```diff
@@ -85,12 +85,16 @@ Document.prototype.createElement = function(tagName) {
 };
 
 global.customElements = global.customElements || {
+  get(tagName) {
+    return customElementsRegistry[tagName];
+  },
+
   define(tagName, proto) {
     tagName = tagName.toLowerCase();
-    if (registeredElements[tagName]) {
-      throw DOMException(`Registration failed for type '${tagName}'. A type with that name is already registered.`);
+    if (customElementsRegistry[tagName]) {
+      throw new Error(`CustomElementRegistryError: '${tagName}' name has already been registered`);
```

Comment:
Out of curiosity, where does this message come from? The previous one came from Chrome's implementation at the time I think.. Current Chrome is
```
"Failed to execute 'define' on 'CustomElementRegistry': this name has already been used with this registry
    at <location>>"
```

Reply:
Nah just a rewording of the chrome error. Can make it exactly the same if you want.